static/templates/settings/playground_settings_admin.hbs

```diff
@@ -6,6 +6,34 @@
             in these external playgrounds.
             {{/tr}}
         </p>
+
+        {{#if is_admin}}
+        <form class="form-horizontal admin-playground-form">
+            <div class="add-new-playground-box grey-box">
+                <div class="new-playground-form wrapper">
+                    <div class="settings-section-title">{{t "Add a new playground" }}</div>
+                    <div class="alert" id="admin-playground-status"></div>
+                    <div class="control-group">
+                        {{!-- TODO: This should have a typeahead suggestion popover --}}
+                        <label for="playground_pygments_language" class="control-label"> {{t "Language" }} </label>
+                        <input type="text" id="playground_pygments_language" name="pygments_language" placeholder="Python" />
+                    </div>
+                    <div class="control-group">
+                        <label for="playground_name" class="control-label"> {{t "Name" }} </label>
+                        <input type="text" id="playground_name" name="name" placeholder="Python3 playground" />
+                    </div>
```

Comment:
If it's convenient, we may want to add "hint" lines to this form explaining what these parameters are.

Reply:
Do you mean hints like these?

![image](https://user-images.githubusercontent.com/30312566/115334783-9d98f180-a1b9-11eb-8339-d236940a2609.png)

Reply:
Yeah (I'm not sure I love the visual style, but it's serviceable and we can always iterate)